pkg/cmd/build/cancel.go

```diff
@@ -14,28 +14,25 @@ import (
 
 func NewCmdBuildCancel(f *factory.Factory) *cobra.Command {
 	var web bool
+	var pipeline string
 
 	cmd := cobra.Command{
 		DisableFlagsInUseLine: true,
-		Use:                   "cancel [number [pipeline]] [flags]",
-		Short:                 "Cancels a build.",
+		Use:                   "cancel <number> [flags]",
```

Comment:
As this is a destructive operation and could cause issues when builds are cancelled whilst running, I think requiring an actual build number here is a good idea. That will mean the user won't unintentionally cancel a `main` branch build which could cause issues with deploys or other things.